addons/barcodes_gs1_nomenclature/models/barcode_nomenclature.py

```diff
@@ -98,7 +98,7 @@ def gs1_decompose_extanded(self, barcode):
         if self.gs1_separator_fnc1:
             separator_group = "(?:%s)?" % self.gs1_separator_fnc1
         # zxing-library patch, removing GS1 identifiers
-        for identifier in [']C1', ']e0', ']d2', ']Q3', ']J1']:
+        for identifier in [']C1', ']e0', ']d2', ']Q3', ']J1', FNC1_CHAR]:
             if barcode.startswith(identifier):
                 barcode = barcode.replace(identifier, '')
```

Comment:
You should precise we only want to replace the first instance of `identifier` because we don't want to remove FNC1 used to separate two GS1 AI (I'm surprised the runbot is green, we lack a test about that)
```suggestion
                barcode = barcode.replace(identifier, '', 1)
```

Reply:
Ok, current tests are green because tests with barcode with FNC1 don't start with FNC1, so they aren't removed.
I think it should be safer to add a test who ensures than:
- We can scan a GS1 barcode starting with a FNC1;
- Same but with another FNC1 between two AI somewhere inside the barcode

Reply:
Thank you for the review!

> You should precise we only want to replace the first instance of `identifier` because we don't want to remove FNC1 used to separate two GS1 AI (I'm surprised the runbot is green, we lack a test about that)

I didn't even consider this but great point, will be sure to amend with this added.

I will look into making the test, just to clarify:
> Same but with another FNC1 between two AI somewhere inside the barcode

Correct behavior should keep the FNC1, because in this case it's not at the start, is that correct?